README.md

```diff
@@ -10,13 +10,62 @@
   <image alt="License" src="https://img.shields.io/github/license/castle/castle-cloudflare-worker-demo"/>
 </div>
 
+## Overview
+
+The Castle Cloudflare Worker allows you to put Castle's risk engine right on the edge, in a Cloudflare worker.
+
+## How it works
+Once you've installed the worker with the default settings, the worker will listen for POSTs to the 
+
+`/users/sign_up`
+
+route.
+
+The POST must include a Castle client_id. Click here to learn more about how to include a Castle client_id in a POST.
```

Comment:
here is missing, I'd also highlight `client_id`

Reply:
good catch